### PR TITLE
Fix command tests

### PR DESCRIPTION
**Pull request by yongpingchen**
Refer #281 
- Implement AliasAction and fix small tests of AliasAction: AliasActionTest
- Fix CommandTest
- Fix CommandFormBuilderTest

### CODE REVIEW

File: thingif/src/main/java/com/kii/thingif/ThingIFAPI.java [hunk 1]
```diff
@@ -202,6 +201,8 @@ public Builder setInstallationID(
          *
          * @param app Kii Cloud Application.
          * @param owner Specify who uses the ThingIFAPI.
+         * @param actionTypes the format of key of actionTypes is "[alias]:[actionName]".
```
Comment (shinsuke-nara) [inline thingif/src/main/java/com/kii/thingif/ThingIFAPI.java]: If alias name is "AirConditionerAlias" and action name is `turnPower`, Which is correct in following 2 strings?

1. `AirConditionerAlias:turnPower`
1. `[AirConditionerAlias]:[turnPower]`

**Comment by shinsuke-nara** [comment]
@yongpingchen Please confirm my comment.

File: thingif/src/main/java/com/kii/thingif/ThingIFAPI.java (hunk 1 above)
Comment (yongpingchen) [inline thingif/src/main/java/com/kii/thingif/ThingIFAPI.java]: first one

Comment (yongpingchen) [inline thingif/src/main/java/com/kii/thingif/ThingIFAPI.java]: @shinsuke-nara I improved the description in commit 34f3940 to make it clear. Can you check it?

**Comment by yongpingchen** [comment]
@shinsuke-nara I improved the description in commit 34f3940 to make it clear. Can you check it?

**Pull request merged**